### PR TITLE
removed references to host (pulse-211)

### PR DESCRIPTION
**Pull request by shashi4569**

### CODE REVIEW

File: system-test/system-test.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ echo "Starting Spark application"
 ./system-test/spark-logging &> system-test/log_files/spark_example.log
 
 echo "Curling the Solr API"
-query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://master3.valhalla.phdata.io:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
+query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://master3.${ZK_HOST}:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
```
Comment (afoerster) [inline system-test/system-test.sh]: Do we need to document in this script that the `ZK_HOST` param is required?

Comment (shashi4569) [reply system-test/system-test.sh]: This is outdated. But added ZK_HOST and other required variables to env.sh.template

File: log-example/src/main/resources/log4j-async.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
         </layout>
     </appender>
     <appender name="http" class="io.phdata.pulse.log.HttpAppender">
-        <param name="Address" value="http://edge2.valhalla.phdata.io:9001/log?application=pulse-test-100"/>
+        <param name="Address" value="http://${ZK_EDGE}:9001/log?application=pulse-test-100"/>
```
Comment (afoerster) [inline log-example/src/main/resources/log4j-async.xml]: This is the log collector host, you can just put something like 'log_collector_hostname'

File: log-example/src/main/resources/log4j.properties [hunk 3]
```diff
@@ -4,7 +4,7 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] %c - %m%n
 
 log4j.appender.http=io.phdata.pulse.log.HttpAppender
-log4j.appender.http.Address=http://edge2.valhalla.phdata.io:9015/v2/events/pulse-test-default
+log4j.appender.http.Address=http://${ZK_EDGE}:9015/v2/events/pulse-test-default
```
Comment (afoerster) [inline log-example/src/main/resources/log4j.properties]: Again a log collector

File: alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineCliParserTest.scala [hunk 4]
```diff
@@ -79,7 +79,7 @@ class AlertEngineCliParserTest extends FunSuite {
     assertResult(49152)(cliParser.smtpPort())
     assertResult("silenced-applications.txt")(cliParser.silencedApplicationsFile())
     assertResult(
-      "master1.valhalla.phdata.io/solr,master2.valhalla.phdata.io/solr,master3.valhalla.phdata.io/solr")(
+      "${HOSTNAME1}/solr,${HOSTNAME2}/solr,${HOSTNAME3}/solr")(
```
Comment (afoerster) [inline alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineCliParserTest.scala]: No need to use (what looks) string interpolation here, just use 'hostname' or the like

File: system-test/log4j.properties [hunk 5]
```diff
@@ -4,7 +4,7 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] (%F:%L) - %m%n
 
 log4j.appender.http=io.phdata.pulse.log.HttpAppender
-log4j.appender.http.Address=http://edge2.valhalla.phdata.io:8214/v2/events/logging-pulse-test
+log4j.appender.http.Address=http://${ZK_EDGE}:8214/v2/events/logging-pulse-test
```
Comment (afoerster) [inline system-test/log4j.properties]: this is a log collector

File: system-test/system-test.sh [hunk 6]
```diff
@@ -38,7 +38,7 @@ echo "Starting Spark application"
 ./system-test/spark-logging &> system-test/log_files/spark_example.log
 
 echo "Curling the Solr API"
-query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://master3.valhalla.phdata.io:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
+query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://${HOSTNAME3}:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
```
Comment (afoerster) [inline system-test/system-test.sh]: Use 'ZOOKEEPER_HOST' instead?

File: log-collector/src/test/scala/io/phdata/pulse/logcollector/LogCollectorCliParserTest.scala [hunk 7]
```diff
@@ -26,13 +26,12 @@ class LogCollectorCliParserTest extends FunSuite {
...
       "9000",
       "--zk-hosts",
-      "master1.valhalla.phdata.io/solr,master2.valhalla.phdata.io/solr,master3.valhalla.phdata.io/solr");
+      "${HOSTNAME1}/solr,${HOSTNAME2}/solr,${HOSTNAME3}/solr");
 
     val cliParser = new LogCollectorCliParser(args)
     assertResult(9000)(cliParser.port())
     assertResult(
-      "master1.valhalla.phdata.io/solr,master2.valhalla.phdata.io/solr,master3.valhalla.phdata.io/solr")(
+      "${HOSTNAME1}/solr,${HOSTNAME2}/solr,${HOSTNAME3}/solr")(
```
Comment (afoerster) [inline log-collector/src/test/scala/io/phdata/pulse/logcollector/LogCollectorCliParserTest.scala]: Use a hard coded string

File: log-appender/README.md [hunk 8]
```diff
@@ -24,7 +24,7 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] (%F:%L) - %m%n
 
 log4j.appender.http=io.phdata.pulse.log.HttpAppender
-log4j.appender.http.Address=http://edge2.valhalla.phdata.io:9015/v2/events/pulse-test-100
+log4j.appender.http.Address=http://<edge node name>:9015/v2/events/pulse-test-100
```
Comment (afoerster) [inline log-appender/README.md]: This is another log collector

**Comment by afoerster** [comment]
@shashi4569 do you plan on revisiting this or should I close the pull request?

**Comment by afoerster** [comment]
Since the documentation has changed I'm closing it.

**Pull request closed without merging**